components/lists/FriendsList.js

```diff
@@ -4,13 +4,12 @@ import {
 } from 'react-native';
 import FriendsCard from '../cards/FriendsCard';
 
-export default function FriendsList({ friends, handleSendMessage }) {
-  //console.log(handleSendMessage(''));
+export default function FriendsList({ friendsList, handleSendMessage }) {
   return (
     <FlatList
-      data={friends}
-      renderItem={({item}) => (<FriendsCard item={item} handleSendMessage={handleSendMessage} />)}
-      keyExtractor={(item) => item.id.toString()}
+      data={friendsList}
+      renderItem={({ item }) => (<FriendsCard item={item} handleSendMessage={handleSendMessage} />)}
```

Comment:
should this send a message, or should it navigate to that friend's profile? I feel like the latter is more intuitive. And Then you should be able to start a chat from a friends' page maybe?